src/workerd/io/compatibility-date.capnp

```diff
@@ -2,6 +2,9 @@
 # Licensed under the Apache 2.0 license found in the LICENSE file or at:
 #     https://opensource.org/licenses/Apache-2.0
 
+# Changes to this file must be reflected in the official Cloudflare docs:
```

Comment:
https://github.com/cloudflare/workerd/pull/741 added a [docs/api-update.md](https://github.com/cloudflare/workerd/blob/main/docs/api-updates.md). The comment here in `compatibility-date.capnp` could just point to that to provide details on the workflow.